flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTestBase.java

```diff
@@ -121,10 +122,13 @@ public static void shutDownServices() throws Exception {
 
 	protected static Configuration getFlinkConfiguration() {
 		Configuration flinkConfig = new Configuration();
+		flinkConfig.setString(AkkaOptions.WATCH_HEARTBEAT_PAUSE, "5 s");
+		flinkConfig.setString(AkkaOptions.WATCH_HEARTBEAT_INTERVAL, "1 s");
+		flinkConfig.setBoolean(AkkaOptions.LOG_LIFECYCLE_EVENTS, true);
```

Comment:
Can we omit this log?

Reply:
Yes sure, I forgot to drop it, it was only for debug purposes